### ISSUE TITLE
Error 400 for account with two-step verification

### ISSUE THREAD

**Issue by tomcpc** [fazo96/telecommander#3]
I get the error "400 PHONE_PASSWORD_PROTECTED" when I try to log with my account which uses two-step verification, so I guess it is not supported yet, right?

**Comment by fazo96** [comment]
Yeah, it's not! I didn't even know it was a thing lol. I'll add it to the list of stuff I need to do and research how to do it :+1:

I'll leave the issue open so that if someone else has the same error he can find out why.
